src/common/src/array/mod.rs

```diff
@@ -125,10 +125,10 @@ pub trait ArrayBuilder: Send + Sync + Sized + 'static {
 /// `Array` must be built with an `ArrayBuilder`. The array trait provides several
 /// unified interface on an array, like `len`, `value_at` and `iter`.
 ///
-/// The `Builder` associated type is the builder for this array. The `Iter` associated
-/// type is the iterator of this array. And the `RefItem` is the item you could
-/// retrieve from this array.
+/// The `Builder` associated type is the builder for this array.
 ///
+/// The `Iter` associated type is the iterator of this array. And the `RefItem` is
+/// the item you could retrieve from this array.
```

Comment:
Intended change or accidental?

Reply:
As intended, since `Iter` is the valid way to consume arrays. I thought it would be better to put `Iter` and `RefItem` in the same block, instead of `Builder` and `Iter` on the same line, especially for quote `Iter` and EOL.